spec/features/restaurants/create_update_restaurant_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+feature 'creating and updating restaurants' do
+  context 'as an authorized user' do
+    let(:user) { FactoryGirl.create(:user) }
+    let(:restaurant) { FactoryGirl.create(:restaurant) }
```

Comment:
use `build` instead of `create` if you want to use the restaurant.name, address etc. below. Otherwise, `create` saves the object to the database before you go through the `fill_in` stuff.